chromium_src/chrome/browser/ui/views/omnibox/omnibox_view_views.h

```diff
@@ -0,0 +1,19 @@
+/* Copyright (c) 2022 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
```

Comment:
FYI, we've added a check in CI that looks for an HTTPS URL here and so you'll probably need to change that to https://mozilla.org/MPL/2.0/ before merging.

Reply:
done